android/src/main/java/co/nimblehq/ic/kmm/suv/android/di/MainModule.kt

```diff
@@ -2,11 +2,18 @@ package co.nimblehq.ic.kmm.suv.android.di
 
 import co.nimblehq.ic.kmm.suv.android.ui.screens.home.HomeViewModel
 import co.nimblehq.ic.kmm.suv.android.ui.screens.surveydetail.SurveyDetailViewModel
+import co.nimblehq.ic.kmm.suv.android.ui.screens.surveyquestions.SurveyQuestionsViewModel
 import org.koin.androidx.viewmodel.dsl.viewModel
 import org.koin.androidx.viewmodel.dsl.viewModelOf
 import org.koin.dsl.module
 
 val mainModule = module {
     viewModelOf(::HomeViewModel)
     viewModel { parameters -> SurveyDetailViewModel(survey = parameters.get()) }
+    viewModel { parameters ->
+        SurveyQuestionsViewModel(
+            surveyQuestionsArgument = parameters.get(),
+            get()
+        )
+    }
```

Comment:
@suho Please update this implementation to get rid of argument dependency on ViewModel 😄